clamav/client/model.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed
+# under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from model import ModelBase
+
+
+class ClamAVInfo(ModelBase):
+    '''Thin wrapper around the information ClamAV returns about itself
+    '''
+    def max_scan_size(self):
+        return self.raw['maxScanSize']
+
+    def signature_timestamp(self):
+        return self.raw['signatureTimestamp']
+
+    def engine_version(self):
+        return self.raw['engineVersion']
+
+
+class ClamAVMonitoringInfo(ModelBase):
+    '''Encapsules the monitoring information returned by ClamAV
```

Comment:
please be consistent w/ docstrings, at least in one module. technical documentation is not about variety. either stick /w `thin wrapper`, or `encapsules`.

Or just omit this boiler plate entirely - the type names are expressive enough.